src/store/user/user.reducer.js

```diff
@@ -20,6 +20,12 @@ export default function userReducer(state = initialState, action) {
                 profilePic: action.payload.profilePic,
             };
         }
+        case RESET_USER_DATA: {
+            return {
+                state: undefined,
+            };
+        }
+
```

Comment:
This should also be just an empty object. A state object with a state property set to undefined doesn't really make sense